crystal-map-connector/src/main/kotlin/ch/ergon/dope/CrystalMapAdapter.kt

```diff
@@ -26,25 +26,25 @@ fun CMType.asField(reference: String = path): Field<out ValidType> = Field(
     reference,
 )
 
-@JvmName("asNumberField")
-fun CMField<out Number>.asField(reference: String = path): Field<NumberType> = Field(name, reference)
+@JvmName("toDopeNumberField")
+fun CMField<out Number>.toDopeField(reference: String = path): Field<NumberType> = Field(name, reference)
 
-@JvmName("asStringField")
-fun CMField<String>.asField(reference: String = path): Field<StringType> = Field(name, reference)
+@JvmName("toDopeStringField")
+fun CMField<String>.toDopeField(reference: String = path): Field<StringType> = Field(name, reference)
 
-@JvmName("asBooleanField")
-fun CMField<Boolean>.asField(reference: String = path): Field<BooleanType> = Field(name, reference)
+@JvmName("toDopeBooleanField")
+fun CMField<Boolean>.toDopeField(reference: String = path): Field<BooleanType> = Field(name, reference)
 
-@JvmName("asNumberArrayField")
-fun CMList<out Number>.asArrayField(): Field<ArrayType<NumberType>> = Field(name, path)
+@JvmName("toDopeNumberArrayField")
+fun CMList<out Number>.toDopeArrayField(): Field<ArrayType<NumberType>> = Field(name, path)
 
-@JvmName("asStringArrayField")
-fun CMList<String>.asArrayField(): Field<ArrayType<StringType>> = Field(name, path)
+@JvmName("toDopeStringArrayField")
+fun CMList<String>.toDopeArrayField(): Field<ArrayType<StringType>> = Field(name, path)
 
-@JvmName("asBooleanArrayField")
-fun CMList<Boolean>.asArrayField(): Field<ArrayType<BooleanType>> = Field(name, path)
+@JvmName("toDopeBooleanArrayField")
+fun CMList<Boolean>.toDopeArrayField(): Field<ArrayType<BooleanType>> = Field(name, path)
 
-fun CMList<out Any>.asArrayField(): Field<ArrayType<ValidType>> = Field(name, path)
+fun CMList<out Any>.toDopeArrayField(): Field<ArrayType<ValidType>> = Field(name, path)
 
 // TODO: DOPE-192
-fun CMObjectList<Schema>.asSchemaArray() = DopeSchemaArray(element, formatPathToQueryString(name, path))
+fun CMObjectList<Schema>.toDopeSchemaArray() = DopeSchemaArray(element, formatPathToQueryString(name, path))
```

Comment:
I don't like the class name `DopeSchemaArray`, but the rename makes sense given the name.

Reply:
Change the name to `toDopeType`.

Reply:
done